include/play.js

```diff
@@ -1,187 +1,201 @@
-          if (!canModifyQueue(member) || queue.volume == 0) return;
-          if (queue.volume - 10 <= 0) queue.volume = 0;
-          else queue.volume = queue.volume - 10;
-          queue.connection.dispatcher.setVolumeLogarithmic(queue.volume / 100);
-          queue.textChannel
-            .send(`${user} 🔉 decreased the volume, the volume is now ${queue.volume}%`)
-            .catch(console.error);
-          break;
-
-        case "🔊":
-          reaction.users.remove(user).catch(console.error);
-          if (!canModifyQueue(member) || queue.volume == 100) return;
-          if (queue.volume + 10 >= 100) queue.volume = 100;
-          else queue.volume = queue.volume + 10;
-          queue.connection.dispatcher.setVolumeLogarithmic(queue.volume / 100);
-          queue.textChannel
-            .send(`${user} 🔊 increased the volume, the volume is now ${queue.volume}%`)
-            .catch(console.error);
-          break;
-
-        case "🔁":
-          reaction.users.remove(user).catch(console.error);
-          if (!canModifyQueue(member)) return;
-          queue.loop = !queue.loop;
-          queue.textChannel.send(`Loop is now ${queue.loop ? "**on**" : "**off**"}`).catch(console.error);
-          break;
-
-        case "⏹":
-          reaction.users.remove(user).catch(console.error);
-          if (!canModifyQueue(member)) return;
-          queue.songs = [];
-          queue.textChannel.send(`${user} ⏹ stopped the music!`).catch(console.error);
-          try {
-            queue.connection.dispatcher.end();
-          } catch (error) {
-            console.error(error);
-            queue.connection.disconnect();
-          }
-          collector.stop();
-          break;
-
-        default:
-          reaction.users.remove(user).catch(console.error);
-          break;
-      }
-    });
-
-    collector.on("end", () => {
-      playingMessage.reactions.removeAll().catch(console.error);
-      if (PRUNING == "true" && playingMessage && !playingMessage.deleted) {
-        playingMessage.delete({ timeout: 3000 }).catch(console.error);
-      }
-    });
-  }
-};
+const ytdlDiscord = require("ytdl-core-discord");
+const scdl = require("soundcloud-downloader");
+const { canModifyQueue, STAY_TIME, LOCALE } = require("../util/EvobotUtil");
+const i18n = require("i18n");
+i18n.setLocale(LOCALE);
+
+module.exports = {
+  async play(song, message) {
+    const { PRUNING, SOUNDCLOUD_CLIENT_ID } = require("../util/EvobotUtil");
+
+    const queue = message.client.queue.get(message.guild.id);
+
+    if (!song) {
+      setTimeout(function () {
+        if (queue.connection.dispatcher && message.guild.me.voice.channel) return;
+        queue.channel.leave();
+        queue.textChannel.send(i18n.__("play.leaveChannel"));
+      }, STAY_TIME * 1000);
+      return queue.textChannel.send(i18n.__("play.queueEnded")).catch(console.error);
+      return message.client.queue.delete(message.guild.id);
```

Comment:
@eritislami Sure! i added translation for 'Leaving the channel...' which related to #351.

But, It seems 'message.client.queue.delete(message.guild.id)' is unreachable due to `return` of L19, 

So, how about modifying it as below?

```
if (!song) {
      setTimeout(function () {
        if (queue.connection.dispatcher && message.guild.me.voice.channel) return;
        queue.channel.leave();
        queue.textChannel.send(i18n.__("play.leaveChannel"));
      }, STAY_TIME * 1000);
      message.client.queue.delete(message.guild.id);
      return queue.textChannel.send(i18n.__("play.queueEnded")).catch(console.error);
    }
```